src/bot.js

```diff
@@ -12,13 +12,18 @@ const config = JSON.parse(
 bot.config = config;
 bot.commands = new Commands(bot);
 bot.logger = new Logger();
-bot.blacklist = new Discord.Collection();
```

Comment:
My actual goal with this was to add a list of users that are abusing the bot so they can't use it, but this is also a great idea!